scripts/test/Muon/muon_context_test.py

```diff
@@ -12,13 +12,26 @@
 
 from mantid.api import AnalysisDataService, FileFinder
 from mantid import ConfigService
-from mantid.simpleapi import CreateWorkspace
+#from mantid.simpleapi import CreateWorkspace
```

Comment:
If the headers are commented out can we delete them?

Reply:
```suggestion
```